workarea-product_badges.gemspec

```diff
@@ -15,5 +15,5 @@ Gem::Specification.new do |s|
 
   s.license = 'Business Software License'
 
-  s.add_dependency "workarea", "~> 3.x", ">= 3.4.x"
+  s.add_dependency "workarea", ">= 3.x"
```

Comment:
So changing this means it's compatible for all Workarea greater than v3, previously it was all Workarea greater than v3.4. Doesn't seem like the change we want?